agent/installer/bundle_builder/ingredients/deb/Dockerfile

```diff
@@ -12,8 +12,8 @@ ARG BASE_IMAGE=ubuntu:18.04
 FROM $BASE_IMAGE as build
```

Comment:
The `$BASE_IMAGE` still uses `ubuntu:18.04`. Is it ok to use `ubuntu:20.04` as this is the supported host os for BYOH?

Reply:
Done. Thanks